packages/rum-core/src/domain/rumEventsCollection/view/trackViewMetrics.spec.ts

```diff
@@ -67,7 +54,7 @@ describe('rum track view metrics', () => {
       .withFakeLocation('/foo')
       .beforeBuild(({ location, lifeCycle }) => {
         lifeCycle.subscribe(LifeCycleEventType.VIEW_UPDATED, handler)
-        return trackViews(location, lifeCycle)
+        return trackViews(location, lifeCycle, configuration as Configuration)
```

Comment:
The `.beforeBuild(({ configuration })` argument should be used instead of an external configuration constant. You can use `.withConfiguration({ isEnabled: () => false })`